gui/GameConqueror.py

```diff
@@ -23,12 +23,11 @@
 
 import sys
 import os
-import re
+import argparse
 import struct
 import tempfile
 import platform
 import threading
-import time
```

Comment:
My fault!
Yes, my linter complains and searching for `time.` yields no results.